### PR TITLE
Problem ActiveField Init.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Fixed issues  | (#5)

### CODE REVIEW

**Comment by samdark** [comment]
Merged. Thanks!

File: src/ActiveField.php [hunk 1]
```diff
@@ -150,11 +149,13 @@ class ActiveField extends \yii\widgets\ActiveField implements Initiable
...
     /**
-     * {@inheritdoc}
+     * Constructor for activefield
      */
-    public function init(): void
+    public function __construct(array $config = [])
     {
-        AbstractContainer::configure($this, $this->createLayoutConfig());
+     	if (!empty($config)) {
+        	AbstractContainer::configure($this, $this->createLayoutConfig());
```
Comment (machour) [inline src/ActiveField.php]: What is $config purpose here? This looks off.

Comment (samdark) [reply src/ActiveField.php]: https://github.com/yiisoft/yii-bootstrap4/commit/27744ee4d9cc5f52e0836a6e0fad7288752bca44

Comment (terabytesoftw) [reply src/ActiveField.php]: @samdark could reverse the commit to the confirmation of @machour https://github.com/yiisoft/yii-bootstrap4/commit/c67c40ace499b6957ddeb5d5f7ea9ced84851b84, my pr it breaks all the tests.

**Comment by machour** [comment]
@terabytesoftw I just reverted it.

Could you open a new issue if you're facing problems again, and provide a little repository that reproduces the problem? Thank you 🙏

**Pull request merged**